ejmask-extensions/src/test/java/com/ebay/ejmask/extenstion/builder/json/JsonBooleanFieldPatternBuilderTest.java

```diff
@@ -0,0 +1,160 @@
+package com.ebay.ejmask.extenstion.builder.json;
+
+import com.ebay.ejmask.api.PatternEntity;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+/**
+ * @author fsun1
+ */
+public class JsonBooleanFieldPatternBuilderTest {
+
+    public static final JsonBooleanFieldPatternBuilder instance = new JsonBooleanFieldPatternBuilder();
+    public static final String[] fieldNames = new String[]{"firstName", "lastName", "number", "boolValue"};
+
+
+    /**
+     * Test of buildPattern method, of class JsonFieldPatternBuilder.
+     */
+    @Test
+    public void testBuildPattern() {
+        int visibleCharacters = 12;
+        String result = instance.buildPattern(visibleCharacters, fieldNames);
+        Assertions.assertEquals("\\\"(firstName|lastName|number|boolValue)(\\\\*\\\"\\s*:\\s*\\\\*)(\\b(true|TRUE|True|false|FALSE|False)\\b)([^\\\"]{1,3})[^\\\"]*(\\\\?\\\"|)", result);
+    }
```

Comment:
which practice do we use for this project?  

```java
class XX {

    void testXXX
    }
```
or  

```java
// more like JUnit4 style
public class XXTest{
    public void testXXX() {
    }
}
```

### reference
https://junit.org/junit5/docs/current/user-guide/#writing-tests

Reply:
We are using Junit5, but still follow public test class style. To keep it constant for now.
We can create a subsequent PR to make this change for all the tests.